cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -268,25 +267,21 @@ instance FromJSON PartialNodeConfiguration where
 
       pure PartialNodeConfiguration {
              pncProtocolConfig
-           , pncSocketPath
+           , pncSocketConfig = PartialSocketConfig pncSocketPath mempty mempty mempty
            , pncDiffusionMode
            , pncSnapshotInterval
            , pncTestEnableDevelopmentNetworkProtocols
            , pncMaxConcurrencyBulkSync
            , pncMaxConcurrencyDeadline
-           , pncLoggingSwitch
+           , pncLoggingSwitch = Last $ Just pncLoggingSwitch'
```

Comment:
We should be able to use `pncLoggingSwitch` if `Last $ Just` is moved back to its original position.

Reply:
In the new version their is an intermediate use of pncLoggingSwitch' as a bool, that's the cause of the move.